projects/core/src/site-context/store/effects/base-site.effect.ts

```diff
@@ -6,6 +6,9 @@ import { makeErrorSerializable } from '../../../util/serialization-utils';
 import { SiteConnector } from '../../connectors/site.connector';
 import { SiteContextActions } from '../actions/index';
 
+/**
+ * @deprecated since 3.0, it will not be used
```

Comment:
I'm not sure if we should deprecate this effect. 

In an ideal world, we'd know the baseSite UID upfront, and would then load additional details when needed, could be even scoped (similar to product scopes). We currently don't have that API on OCC, but going forward we would go by the facade->action->effect->connect->adapter. This is where I'd expect this effect to be used.